html-practice/facebook-mock-up.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<style>
+		html {
+			background-color: #e9eaed;
+			height: 100%;
+			width: 100%;
+		}
+		body {
+			width: 100%;
+			height: 800px;
+			margin: 0;
+		}
+		#Facebook-menu-bar {
+			background-color: #3a5795;
+			color: white;
+			position: relative;
+			top: 0;
+			height: 45px;
+			width: 100%;
+			margin: 0;
+		}
+		#search-bar {
+			position: relative;
+			top: 7px;
+			left: 225px;
+			width: 450px;
+			height: 15px;
+			padding: 5px;
+			border: 1px solid black;
+			border-radius: 5px;
+		}
+		#content {
+			width: 500px;
+			height: 100%;
+			margin: 0 auto;
+			position: fixed;
+			left: 400px;
+		}
+		.status-box {
+			background-color: white;
+			border: 1px solid #e9eaed;
+			margin: 10px 0;
+			height: 140px;
+		}
+		.status-box-options {
+			color: blue;
+			border-bottom: 1px solid #e9eaed;
+			list-style-type: none;
+			display: inline-block;
+			width: 100%;
+			padding: 0;
+			margin: 0;
+		}
+		.status-box-options span {
+			float: left;
+			padding: 5px;
+			border-right: 1px solid #e9eaed;
+			color: #3a5795;
+			font-weight: bold;
+		}
+		.status-update-input img {
+			height: 50px;
+			width: 50px;
+			float: left;
+			margin-left: 5px;
+		}
+		.status-update-input input{
+			margin-left: 5px;
+			width: 80%;
+			padding: 15px;
+			border: none;
+			height: 40px;
+			float: left;
+		}
+		.status-update-options {
+			display: inline-block;
+			list-style-type: none;
+			width: 100%;
+			margin: 0;
+			padding: 0;
+			border-top: 1px solid #e9eaed;
+		}
+		.status-update-options span {
+			float: left;
+			padding: 5px;
+		}
+		.status-update-options input {
+			float: right;
+			margin: 5px;
+		}
+		#sample-status {
+			background-color: white;
+			border: 1px solid #e9eaed;
+			margin: 10px 0;
+			height: 120px;
+			padding-top: 10px;
+		}
+		#sample-status-title {
+			color: #3a5795;
+			margin-left: 10px;
+			font-weight: bold;
+			font-family: sans-serif;
+			font-size: 14px;
+		}
+		#sample-status img {
+			height: 50px;
+			width: 50px;
+			float: left;
+			margin-left: 5px;
+		}
+		#sample-status p {
+			position: relative;
+			top: 25px;
+			text-align: left;
+			padding: 10px 0;
+			margin: 10px;
+		}
+		.sample-status-options {
+			display: inline-block;
+			list-style-type: none;
+			width: 100%;
+			margin: 10px 0;
+			padding: 0;
+			border-top: 1px solid #e9eaed;
+		}
+		.sample-status-options span {
+			float: left;
+			padding: 5px;
+		}
+		#ad-content {
+
+		}
+		</style>
+	</head>
+	<body>
+		<header id="Facebook-menu-bar">
+			<input type="text" id="search-bar" placeholder="Search Facebook">
+		</header>
+		<main id="content">
+			<div class="status-box">
+				<ul class="status-box-options">
+					<li><span>Update Status</span></li>
```

Comment:
If you remove these `<span>`s and change the rule to apply to the `li` elements instead of `span`s, would it look the same?

If so, that would be a tiny way to simplify this section.